src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -1365,6 +1388,8 @@ def setNumberOfCores(self, cores, nStreams):
             taskStreams = nStreams
 
         if taskCores:
+            # if # of cores is changed, then also change the MaxVSize values
+            self.setMaxVSize(taskCores * 20 * 1024)  # 20 GiB per core
```

Comment:
same as above  - shoudn't be hard coded?

Reply:
Same as above, hardcode to 1TB would also work, but 20GB per core is also something ridiculously high and should also work. They are all equivalent to "no limit".